azkaban-common/src/main/java/azkaban/project/ProjectManager.java

```diff
@@ -196,61 +174,44 @@ public Props getProps() {
   }
 
     /**
-     * Checks if a project is active using project_name
-     *
-     * @param name
-     */
-    public Boolean isActiveProject(String name) {
-        return projectsByName.containsKey(name);
-    }
-
-    /**
      * Checks if a project is active using project_id
      *
-     * @param name
+     * @param id
      */
     public Boolean isActiveProject(int id) {
```

Comment:
It is used in handlePurgeProject() which will do the actual cleanup of the projects. If the project is still active, then it will return some error saying the project should be deleted before purging.